graphrag/llm/openai/utils.py

```diff
@@ -87,17 +89,51 @@ def get_completion_llm_args(
     }
 
 
-def try_parse_json_object(input: str) -> dict:
-    """Generate JSON-string output using best-attempt prompting & parsing techniques."""
+def try_parse_json_object(input: str) -> tuple[str, dict]:
```

Comment:
@AlonsoGuevara there is one edge case here that may explain @xxll88's comment about the significant increase in indexing/query time. Assume `input` was json-formatted correctly by the LLM - the logic in this function would still attempt to call `.replace()` multiple times on the string when it's not necessary.

Perhaps there should be a try-except call made to `json.loads(...)` at the top of this function to first check whether any further string operations are needed. Arguably, this *edge case* is the most generic case encountered when using a model that guarantee's json output. It turns into a lot of unnecessary string operations.

Reply:
Thanks for your comments @jgbradley1 and @xxll88 
Addressing this on #834